WordPress/Tests/Security/NonceVerificationUnitTest.inc

```diff
@@ -160,3 +160,21 @@ function foo_8() {
 	sanitize_pc( $_POST['bar'] ); // Bad.
 	my_nonce_check( sanitize_twitter( $_POST['tweet'] ) ); // Bad.
 }
+
+/*
+ * Using a superglobal in a is_...() function is ok as long as it is not otherwise
```

Comment:
Can this "not ... without" double-negative be rewritten as a positive, to make it easier to understand?

Reply:
I've adjusted the comment.